app/views/partials/conversationView.scala.html

```diff
@@ -32,6 +32,10 @@
      font-size: 16px;
      padding-top: 2px;
 }
+
+.breakLong  {
```

Comment:
class names at HMRC have the standard of using kebab-casing as opposed to camel casing.